velox/exec/HashBuild.cpp

```diff
@@ -66,39 +66,70 @@ HashBuild::HashBuild(
     DriverCtx* driverCtx,
     std::shared_ptr<const core::HashJoinNode> joinNode)
     : Operator(driverCtx, nullptr, operatorId, joinNode->id(), "HashBuild"),
-      joinType_{joinNode->joinType()},
+      joinNode_(std::move(joinNode)),
+      joinType_{joinNode_->joinType()},
+      joinBride_(operatorCtx_->task()->getHashJoinBridge(
+          operatorCtx_->driverCtx()->splitGroupId,
+          planNodeId())),
       mappedMemory_(operatorCtx_->mappedMemory()) {
-  auto type = joinNode->sources()[1]->outputType();
+  auto outputType = joinNode_->sources()[1]->outputType();
```

Comment:
Is it possible that `joinNode_->sources().size() < 1` when we construct the plans?

Reply:
I guess now. This change doesn't change this logic as well. Thanks!